src/core/qgsfields.h

```diff
@@ -182,9 +193,16 @@ class CORE_EXPORT  QgsFields
     % End
 #endif
 
-    //! Gets field at particular index (must be in range 0..N-1)
+#ifndef SIP_RUN
+    //! Returns the field at particular index (must be in range 0..N-1)
+    QgsField at( int i ) const SIP_FACTORY;
+#else
+
+    /**
+     * Returns the field at particular index (must be in range 0..N-1).
+     * \throws KeyError if no field exists at the specified index
```

Comment:
I'm wondering if it wouldn't be better to just surround the line with the ``\throws`` with #ifdef SIP_RUN / #endif (here and everwhere else). This would reduce the amount of copy&paste

Reply:
Good idea, done

Reply:
Hmm... doxygen says no unfortunately! It treats # inside a doxygen block as a link request